consensus/src/epoch_manager.rs

```diff
@@ -225,6 +228,11 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
             dag_config,
             payload_manager: Arc::new(PayloadManager::DirectMempool),
             rand_storage,
+            proof_cache: Cache::builder()
+                .max_capacity(10_000)
```

Comment:
Do we need to make these configurable? May be we need this to be bigger for higher TPS?

Reply:
Sure I'll make this configurable. For now, 10K should be enough. It basically needs to be larger than the number proofs that have not been proposed yet.